src/Wallabag/CoreBundle/Resources/translations/messages.es.yml

```diff
@@ -19,9 +19,9 @@ menu:
         unread: 'Sin leer'
         starred: 'Favoritos'
         archive: 'Archivo'
-        all_articles: 'Todos los artículos'
+        all_articles: 'Todos artículos'
         config: 'Configuración'
-        tags: 'Tags'
+        tags: 'Etiquetas'
```

Comment:
I'll check on the *etiquetas *when I get another chance in a couple of
hours.

Thanks

On Mon, Apr 4, 2016 at 9:27 AM, Jami Higashi Lee <jhigashilee@gmail.com>
wrote:

> Yes, and now I'm catching on that I need to erase the # and space before
> each line. Thanks for teaching me that...
>
> On Mon, Apr 4, 2016 at 5:17 AM, Jeremy Benoist <notifications@github.com>
> wrote:
>
>> In src/Wallabag/CoreBundle/Resources/translations/messages.es.yml
>> <https://github.com/wallabag/wallabag/pull/1841#discussion_r58392906>:
>>
>> >          config: 'Configuración'
>> > -        tags: 'Tags'
>> > +        tags: 'Etiquetas'
>>
>> It *tags* become *etiquetas* could you check for other *tag* translation
>> in that file too?
>>
>> —
>> You are receiving this because you modified the open/close state.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/wallabag/wallabag/pull/1841/files/83546fc1cd422766651f8ffd6ec9125e7322fe1d#r58392906>
>>
>
>